<launch>

    <group ns="vl1">

        <param name="calibration" value="$(find vl_lidar_driver)/cfg/VLP16db.yaml"/>
        <param name="max_range" value="100.0"/>
        <param name="min_range" value="0.3"/>
        <param name="view_direction" value="0.0"/>
        <param name="view_width" value="0.0"/>
        <param name="model" value="VLP16"/>
        <param name="read_fast" value="false"/>
        <param name="read_once" value="false"/>
        <param name="repeat_delay" value="0.0"/>
        <param name="rpm" value="600.0"/>
        <param name="cut_angle" value="-0.01"/>

        <param name="frame_id" value="odom"/>
        <param name="npackets" value="76"/><!--75-->
        <param name="begin_cut_angle" value="0"/>
        <param name="end_cut_angle" value="0"/>
        <param name="raw_data_topic_" value="/velodyne1/velodyne_points"/>
        <param name="data_set_topic_" value="/drivers/velodyne1/lidar_points"/>

        <param name="xmin" value="-1.6"/>
        <param name="xmax" value="1.6"/>
        <param name="ymin" value="-7.6"/>
        <param name="ymax" value="7.6"/>

        <param name="log_dir" value="/work/log/"/>
        <param name="sensor_name" value="velodyne"/>
        <param name="device_name" value="lidar1"/>
        <param name="packet_size" value="1206"/>
	    
        <param name="device_ip" value="192.168.2.201"/><!--192.168.2.201-->
        <param name="intput_port" value="2369"/>

        <param name="pub_raw_data" value="1"/>

        <param name="Matrix4f_1" value="0.0846055, -0.996414, 0.00072535, 1.37788, 0.99641, 0.0846034, -0.00313076, 7.24925, 0.00305814, 0.000987549,0.999996, -0.899701, 0, 0, 0, 1"/>
 
        <node pkg="vl_lidar_driver" name="vl_lidar_node_201" type="vl_lidar_node" />
    </group>

</launch>
